service/src/main/resources/application.yml

```diff
@@ -62,6 +62,17 @@ drshub:
   bardEventLoggingEnabled: ${BARD_EVENT_LOGGING_ENABLED:true}
   trackInMixpanel: false
 
+management:
+  # For Micrometer metrics gathering
+  endpoints:
+    web:
+      exposure:
+        # Expose all management endpoints in Spring Boot
+        include: "*"
+  server:
+    # Expose metrics on a different port than our app so that they aren't exposed with other endpoints
+    port: 9098
```

Comment:
Slack context starting here: https://broadinstitute.slack.com/archives/CADM7MZ35/p1670510559479759?thread_ts=1670268800.876879&cid=CADM7MZ35

Reply:
I guess that's probably safer than just path level firewalling via the proxy